bindings/java/Swig/post-build.cmd

```diff
@@ -30,12 +31,18 @@ if "%is_gpu%" == "true" (
   )
 )
 
+if "%is_debug%" == "true" (
+  set opencv_dll=opencv_world310d.dll
+) else (
+  set opencv_dll=opencv_world310.dll
+)
+
 for %%x in (libiomp5md.dll mklml.dll Cntk.Math-%version%.dll Cntk.PerformanceProfiler-%version%.dll Cntk.Core-%version%.dll Cntk.Core.JavaBinding-%version%.dll) do (
   copy "%output_dir%/%%x" ".\com\microsoft\CNTK\lib\windows\%%x" 
   echo %%x>> .\com\microsoft\CNTK\lib\windows\NATIVE_MANIFEST
 )
 
-for %%x in (mkldnn.dll) do (
+for %%x in (zip.dll zlib.dll opencv_dll mkldnn.dll) do (
```

Comment:
opencv_dll - Should it be %opencv_dll% ?

Reply:
Thanks for catching that!